MyCards/MyCards/PreviewView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  PreviewView.swift
+//  MyCards
+//
+//  Created by Maciej Piotrowski on 23/11/16.
+//  Copyright © 2016 Maciej Piotrowski. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+class PreviewView: UIView {
+
+    let captureButton = PhotoCameraButton(frame: .zero)
+    let closeButton = CloseButton(frame: .zero)
```

Comment:
what do you mean? should I add property type to the definition ?
` let captureButton: PhotoCameraButton = PhotoCameraButton(frame: .zero)`